packaging/debs/apt-repository/distributions

```diff
@@ -2,6 +2,6 @@ Origin: RabbitMQ
 Label: RabbitMQ Repository for Debian / Ubuntu etc
 Suite: testing
 Codename: kitten
-Architectures: AVR32 alpha amd64 arm armel armhf arm64 hppa hurd-i386 i386 ia64 kfreebsd-amd64 kfreebsd-i386 m32 m68k mips mipsel netbsd-alpha netbsd-i386 powerpc s390 s390x sh sparc source
+Architectures: AVR32 alpha amd64 arm arm64 armeb armel armhf avr32 hppa hurd-i386 i386 ia64 kfreebsd-amd64 kfreebsd-i386 m32 m32r m68k mips mips64 mips64el mips64r6 mips64r6el mipsel mipsr6 mipsr6el netbsd-alpha netbsd-i386 nios2 or1k powerpc powerpcel ppc64 ppc64el s390 s390x sh sh3 sh3eb sh4 sh4eb sparc sparc64 tilegx source
```

Comment:
Do you know the difference between `AVR32` and `avr32`? It feels like the uppercase one (the first entry in the list) is incorrect.

Reply:
Yeah, I think you're right -- want me to remove the uppercase version, since `dpkg` should never use it according to the `dpkg` source?

Reply:
@tianon please do.